src/wp-admin/widgets-form.php

```diff
@@ -11,14 +11,6 @@
 	die( '-1' );
 }
 
-$widgets_access = get_user_setting( 'widgets_access' );
-if ( isset( $_GET['widgets-access'] ) ) {
-	check_admin_referer( 'widgets-access' );
-
-	$widgets_access = 'on' === $_GET['widgets-access'] ? 'on' : 'off';
-	set_user_setting( 'widgets_access', $widgets_access );
-}
-
 if ( 'on' === $widgets_access ) {
```

Comment:
`$widgets_access` will be undefined here now.

Remind me: what does `?widgets-acess` do? Is it "accessibility mode"? Maybe we don't need it in the new screen.

Reply:
I dug into it and yes it's accessibility mode which we don't have on the new screen, so let's not move this check.